### PR TITLE
Feature/authorize net prod mode

### PR DESCRIPTION
**Pull request by ps2-controller**
maps to this
https://github.com/canopy1/canopy-api/pull/1468

### CODE REVIEW

File: schemas/OrgPaymentProcessorConfig.json [hunk 1]
```diff
@@ -40,6 +40,12 @@
           "nullable": true,
           "example": "346HZ32z3fP4hTG2",
           "description": "Your Authorize.net merchant transaction key."
+        },
+        "is_prod": {
+          "type": "boolean",
+          "nullable": true,
+          "example": "346HZ32z3fP4hTG2",
```
Comment (phorical) [inline schemas/OrgPaymentProcessorConfig.json:47]: Can we change this example to be either true/false?

Comment (ps2-controller) [reply schemas/OrgPaymentProcessorConfig.json:47]: good catch, updated -- thanks!

Comment (phorical) [reply schemas/OrgPaymentProcessorConfig.json:47]: There are two more instances of this example, can you change the other two as well?

Comment (ps2-controller) [reply schemas/OrgPaymentProcessorConfig.json:47]: oh haha I had but forgot to save them so they weren't in the push, done now :) sorry about that

**Review by phorical — APPROVED** [review]

**Review by phorical — APPROVED** [review]

**Pull request merged**